apps/web/src/components/SearchRequestTable/SearchRequestTable.tsx

```diff
@@ -60,16 +61,18 @@ export const SearchRequestTable = ({
             "Title displayed for the search request table edit column.",
         }),
         id: "action", // required when accessor is a function
-        accessor: ({ id, fullName }) =>
+        accessor: (d) => `Action ${d.id}`,
```

Comment:
Is it possible to disable sort for a column?  It's meaningless to sort an action column.

Reply:
Yes! But that causes other issues. I created a ticket for that as well https://github.com/GCTC-NTGC/gc-digital-talent/issues/5642